internal/handlers/handlers.go

```diff
@@ -78,6 +78,9 @@ type Interface interface {
 	// MsgDropDatabase drops production database.
 	MsgDropDatabase(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error)
 
+	// MsgExplain represent explain diagnostic command.
```

Comment:
The same issue as with the help text

Reply:
> The same issue as with the help text

changed to `Display the execution plan.`